Unknown option "--enable-libsvtvp9".

When I build with `./cross_compile_ffmpeg.sh --build-ffmpeg-static=y --build-ffmpeg-shared=y --enable-gpl=n --compiler-flavors=win64 --build-intel-qsv=y --build-amd-amf=y`, an error was occured.

```
config options --pkg-config=pkg-config --pkg-config-flags=--static --extra-version=ffmpeg-windows-build-helpers --enable-version3 --disable-debug --disable-w32threads --arch=x86_64 --target-os=mingw32 --cross-prefix=/home/runner/work/FFmpeg-Auto-Build/FFmpeg-Auto-Build/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/bin/x86_64-w64-mingw32- --enable-libcaca --enable-gray --enable-libtesseract --enable-fontconfig --enable-gmp --enable-gnutls --enable-libass --enable-libbluray --enable-libbs2b --enable-libflite --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libvorbis --enable-libwebp --enable-libzimg --enable-libzvbi --enable-libmysofa --enable-libopenjpeg --enable-libopenh264 --enable-liblensfun --enable-libvmaf --enable-libsrt --enable-demuxer=dash --enable-libxml2 --enable-opengl --enable-libdav1d --enable-cuda-llvm --enable-libsvthevc --enable-libsvtav1 --enable-libsvtvp9 --enable-libaom --enable-libvpx --enable-nvenc --enable-nvdec --extra-libs=-lharfbuzz --extra-libs=-lm --extra-libs=-lpthread --extra-cflags=-DLIBTWOLAME_STATIC --extra-cflags=-DMODPLUG_STATIC --extra-cflags=-DCACA_STATIC --enable-amf --enable-libmfx --enable-avresample --extra-cflags=-mtune=generic --extra-cflags=-O3 --enable-static --disable-shared --prefix=/home/runner/work/FFmpeg-Auto-Build/FFmpeg-Auto-Build/ffmpeg-windows-build-helpers/sandbox/cross_compilers/mingw-w64-x86_64/x86_64-w64-mingw32 ./configure
Unknown option "--enable-libsvtvp9".
See ./configure --help for available options.
failed configure ffmpeg_git_lgpl
##[error]Process completed with exit code 1.
```

But I can build with `./cross_compile_ffmpeg.sh --build-ffmpeg-static=y --build-ffmpeg-shared=y --enable-gpl=n --compiler-flavors=win64 --build-intel-qsv=y --build-amd-amf=y --ffmpeg-git-checkout-version=n4.4`.

How do I fix it?

```
error: can't open patch '../SVT-HEVC_git/ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch': No such file or directory
error: can't open patch '../SVT-VP9_git/ffmpeg_plugin/master-0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch': No such file or directory
configuring ffmpeg-source
...
Unknown option "--enable-libsvthevc".
See ./configure --help for available options.
failed configure ffmpeg-source
##[error]Process completed with exit code 1.
```

build busted since commit d2342d6

@rdp maybe you need to set up github actions to check pull request.

--- a/cross_compile_ffmpeg.sh
+++ b/cross_compile_ffmpeg.sh
@@ -2310,7 +2310,7 @@ build_ffmpeg() {
     #SVT-HEVC
     git apply "../SVT-HEVC_git/ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch"
     git apply "$patch_dir/SVT-HEVC-0002-doc-Add-libsvt_hevc-encoder-docs.patch"  # upstream patch does not apply on current ffmpeg master
-    git apply "../SVT-VP9_git/ffmpeg_plugin/master-0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch"
+    # git apply "../SVT-VP9_git/ffmpeg_plugin/master-0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch"
 
     fi
     if [ "$bits_target" = "32" ]; then
@@ -2337,7 +2337,7 @@ build_ffmpeg() {
     if [ "$bits_target" != "32" ]; then
       config_options+=" --enable-libsvthevc"
       config_options+=" --enable-libsvtav1"
-      config_options+=" --enable-libsvtvp9"
+      # config_options+=" --enable-libsvtvp9"
     fi
 
     #aom must be disabled to use SVT-AV1

